internals-js/src/definitions.ts

```diff
@@ -2319,7 +2319,7 @@ export class EnumType extends BaseNamedType<OutputTypeReferencer, EnumType> {
     }
     const existing = this.value(toAdd.name);
     if (!existing) {
-      this._values.push(toAdd);
+      this._values.add(toAdd);
```

Comment:
An important question with `Map` and `Set` is what you want to happen when you add an element that's already in the container. By default, it won't be bumped up to the most recent / last position, but you can achieve that behavior by deleting the item before adding it (if it already exists).

However, it looks like this code only ever calls `this._values.add` when the name does not yet exist, so using a `Map` for `this._values` should work without the trick of deleting elements.

Reply:
I think the default behavior should be fine.